src/photosui.cs

```diff
@@ -480,4 +481,92 @@ interface PHProjectTypeDescriptionInvalidator
 	interface PHEditingExtensionContext
 	{
 	}
-}
\ No newline at end of file
+
+	interface IPHPickerViewControllerDelegate {}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof(NSObject))]
+	interface PHPickerViewControllerDelegate
+	{
+		[Abstract]
+		[Export ("picker:didFinishPicking:")]
+		void DidFinishPicking (PHPickerViewController picker, PHPickerResult[] results);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (UIViewController))]
+	[DisableDefaultCtor]
+	interface PHPickerViewController
+	{
+		[Export ("configuration", ArgumentSemantic.Copy)]
+		PHPickerConfiguration Configuration { get; }
+
+		[Wrap ("WeakDelegate")]
+		IPHPickerViewControllerDelegate Delegate { get; set; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; set; }
+
+		[Export ("initWithConfiguration:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (PHPickerConfiguration configuration);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface PHPickerConfiguration : NSCopying
+	{
+		[Export ("selectionLimit")]
+		nint SelectionLimit { get; set; }
+
+		[NullAllowed, Export ("filter", ArgumentSemantic.Copy)]
+		PHPickerFilter Filter { get; set; }
+
+		[Export ("initWithPhotoLibrary:")]
+		IntPtr Constructor (PHPhotoLibrary photoLibrary);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PHPickerFilter : NSCopying
```

Comment:
`__attribute__((objc_subclassing_restricted))`
My guess is that we should `[Seal]` the type

Reply:
makes even more sense since it's all static members